### PR TITLE
changes for python3.6

### PR DESCRIPTION
**Pull request by Whadup**
Hi there,
to run on my mac with python 3.6, I had to do these small changes.
apparently, the api for collections has changed from `iteritems()` to `items()` and typechecking for numpy integers is more strict (int divided by 2 gives float)
Hope this helps,
Cheers
Lukas

### CODE REVIEW

**Review by EigenPro — DISMISSED** [review]
Thanks Lukas! This neat modification is compatible with Python 2.7. So I would like to just merge it. But could you revert the change for the assert (L 40) in this commit.? I want to keep this for a while to avoid the relatively disappointing performance on CPU.

**Comment by Whadup** [comment]
I will post another pull request with my changes once I’ve written up my results using your algorithm.

Until then, maybe use this patch?

    From f0713775974831ff502d92c961826124991ca9b1 Mon Sep 17 00:00:00 2001
    From: Lukas Pfahler <lukas.pfahler@tu-dortmund.de>
    Date: Tue, 5 Dec 2017 10:21:04 +0100
    Subject: [PATCH] changes for python3.6
    
    ---
     run_expr.py | 6 +++---
     1 file changed, 3 insertions(+), 3 deletions(-)
    
    diff --git a/run_expr.py b/run_expr.py
    index f2ba50e..c66e35d 100644
    --- a/run_expr.py
    +++ b/run_expr.py
    @@ -128,7 +128,7 @@ trainers['Kernel EigenPro'] = Trainer(model=model,
     # Assemble SGD trainer.
     rff_weights = np.float32(       # for Gaussian kernel
         np.sqrt(2. / (2 * 5 ** 2))  # s = 5
    -    * np.random.randn(D, d/2))
    +    * np.random.randn(D, d>>1))
     input_shape = (D,)
     x = Input(shape=input_shape, dtype='float32', name='feat')
     rf_f = RFF(rff_weights, input_shape=input_shape)
    @@ -159,7 +159,7 @@ trainers['EigenPro with random Fourier feature'] = Trainer(
 	    model=model, x_train = x_train,	x_test=x_test)
     
     # Start training.
    -for name, trainer in trainers.iteritems():
    +for name, trainer in trainers.items():
         print("")
         initial_epoch=0
         np.random.seed(1) # Keras uses numpy random number generator
    -- 
    2.13.1
